.github/workflows/pre-submit.actions.yml

```diff
@@ -516,8 +516,8 @@ jobs:
         with:
           repository: ${{ steps.detect.outputs.repository }}
           ref: ${{ steps.detect.outputs.ref }}
-          builder-ref: "refs/tags/v1.6.0"
-          go-version: "1.21"
+          builder-ref: "refs/tags/v2.1.0"
+          go-version: "1.23.1"
```

Comment:
Should be the latest go 1.24?

Reply:
I decided to keep it the same go version as the particular module being compiled by that script. I know new compilers are backwards-compatible with older modules, but 1.24 is only a single minor version bump, so it didn't seem worth it.